src/BibleWell.App/ViewModels/OpenFileViewModel.cs

```diff
@@ -0,0 +1,55 @@
+using Avalonia.Media.Imaging;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using Microsoft.Maui.Storage;
+
+namespace BibleWell.App.ViewModels;
+
+public partial class OpenFileViewModel : ViewModelBase
+{
+
+    [ObservableProperty]
+    private string? _fileText;
+
+    [ObservableProperty]
+    private Bitmap? _image;
+
+    [ObservableProperty]
+    private bool _isImageVisible;
+
+    [RelayCommand]
+    private async Task OpenFileWithMauiFilePicker(CancellationToken ct)
+    {
+        ErrorMessages?.Clear();
+        try
+        {
+            var file = await FilePicker.PickAsync();
```

Comment:
Okay, I'm technically not wimping out from getting dependency injection to work, because this is literally all that's needed for a FilePicker. Why have a service for this when it's... just this?